x-pack/plugins/cases/server/authorization/authorization.test.ts

```diff
@@ -983,4 +983,202 @@ describe('authorization', () => {
+          privileges: { kibana: [] },
+        }))
+      );
+
+      featuresStart = featuresPluginMock.createStart();
+      featuresStart.getKibanaFeatures.mockReturnValue([feature] as unknown as KibanaFeature[]);
+
+      spacesStart = createSpacesDisabledFeaturesMock();
+
+      auth = await Authorization.create({
+        request,
+        securityAuth: securityStart.authz,
+        spaces: spacesStart,
+        features: featuresStart,
+        auditLogger: new AuthorizationAuditLogger(mockLogger),
+        logger: loggingSystemMock.createLogger(),
+      });
+    });
+
+    it('throws and logs an error when there are no registered owners from plugins and security is enabled', async () => {
+      expect.assertions(2);
+
+      featuresStart.getKibanaFeatures.mockReturnValue([]);
+
+      auth = await Authorization.create({
+        request,
+        securityAuth: securityStart.authz,
+        spaces: spacesStart,
+        features: featuresStart,
+        auditLogger: new AuthorizationAuditLogger(mockLogger),
+        logger: loggingSystemMock.createLogger(),
+      });
+
+      try {
+        await auth.getAuthorizedEntities({
+          entities: [{ id: '1', owner: 'b' }],
+          operation: Operations.bulkGetCases,
+        });
+      } catch (error) {
+        expect(error.message).toBe('Unauthorized to access cases of any owner');
+      }
+
+      expect(mockLogger.log.mock.calls).toMatchInlineSnapshot(`
+        Array [
+          Array [
+            Object {
+              "error": Object {
+                "code": "Error",
+                "message": "Unauthorized to access cases of any owner",
+              },
+              "event": Object {
+                "action": "case_bulk_get",
+                "category": Array [
+                  "database",
+                ],
+                "outcome": "failure",
+                "type": Array [
+                  "access",
+                ],
+              },
+              "message": "Failed attempt to access a cases as any owners",
+            },
+          ],
+        ]
+      `);
+    });
+
+    it('does not throw an error or log when a feature owner exists and security is disabled', async () => {
+      expect.assertions(2);
+
+      auth = await Authorization.create({
+        request,
+        spaces: spacesStart,
+        features: featuresStart,
+        auditLogger: new AuthorizationAuditLogger(mockLogger),
+        logger: loggingSystemMock.createLogger(),
+      });
+
+      const helpersPromise = auth.getAuthorizedEntities({
+        entities: [
+          { id: '1', owner: 'blah' },
+          { id: '2', owner: 'something-else' },
+        ],
+        operation: Operations.bulkGetCases,
+      });
+
+      await expect(helpersPromise).resolves.not.toThrow();
+
+      expect(mockLogger.log.mock.calls).toMatchInlineSnapshot(`Array []`);
+    });
+
+    describe('hasAllRequested: true', () => {
+      it('returns correct entities', async () => {
```

Comment:
To my understanding, yes. If `hasAllRequested === true` then the valid owners are all owners returned by `featuresStart.getKibanaFeatures` and not the ones the user has access (the privlages returned by `checkPrivileges` is an empty array in this test). Then. we filter out. Basically, owner `c` is not a valid one because is not registered as a feature (security) by any plugins